_data/news.yml

```diff
@@ -1,3 +1,7 @@
+- name: Dani Arribas-Bel, SDSC Award
+  date: 2020-10-23
+  summary: Congratulations to <a href="https://github.com/darribas">Dani Arribas-Bel</a> for being named <a href="https://carto.com/blog/announcing-spatial-data-scientist-of-the-year-2020/">Spatial Data Scientist of the Year</a>.
```

Comment:
this deserves an exclamation point, no???

Reply:
Good catch. Added.